front/package.json

```diff
@@ -81,29 +81,29 @@
     "mermaid": "^10.9.0",
     "minimist": "^1.2.8",
     "moment-timezone": "^0.5.43",
-    "next": "^13.5.6",
+    "next": "^14.2.3",
     "next-remove-imports": "^1.0.8",
     "openai": "^4.28.0",
-    "pdfjs-dist": "^4.2.67",
     "pegjs": "^0.10.0",
     "pg": "^8.8.0",
     "pg-hstore": "^2.3.4",
     "pino": "^8.11.0",
     "pino-pretty": "^10.0.0",
-    "react": "^18.2.0",
+    "react": "^18",
```

Comment:
Can we be stricter on react, nodes, tsx and typescript at least?

Reply:
Why? :)